src/asciidoc/docs/guides-general/capture-user-input.adoc

```diff
@@ -0,0 +1,319 @@
+  };
+  ...
+  return (
+    <div className="App">
+      <header className="App-header" style={{ backgroundColor }}>
+        <button onClick={captureRange}>{'.range test'}</button>
+      </header>
+    </div>
+  );
+----
+====
+
+** `.select` – Prompts the user to select from a predefined list of options.
+<<core-frontend-ui-capture#select,API reference>>
++
+.View image and example.
+[%collapsible]
+====
+If the list has 1 to 4 options, the options are presented as radio buttons. +
+If the list has 5 or more options, the list is presented as a dropdown.
+
+image::ui-capture-select.png[UI / Capture .select control,50%,link="{imagesDir}/ui-capture-select.png"]
+
+.Example
+[source,javascript]
+----
+  const captureSelection = async () => {
+    const capturedSelection = await Koji.ui.capture.select(
+      {
+        placeholder : "Choose a mystery prize:",
+        options : [
+          { value : "prizeA", label : "Prize A" },
+          { value : "prizeB", label : "Prize B" },
+          { value : "prizeC", label : "Prize C" },
+          { value : "prizeD", label : "Prize D" },
+          { value : "prizeE", label : "Prize E" }
+        ]
+      }
+    );
+    if (capturedSelection) alert( 'You chose ' + capturedSelection );
+  };
+  ...
+  return (
+    <div className="App">
+      <header className="App-header" style={{ backgroundColor }}>
+        <button onClick={captureSelection}>{'.select test'}</button>
+      </header>
+    </div>
+  );
+----
+====
+
+** `.video` – Prompts the user to select or upload a video file.
+<<core-frontend-ui-capture#video,API reference>>
++
+.View image and example.
+[%collapsible]
+====
+image::ui-capture-video.png[UI / Capture .video control,50%,link="{imagesDir}/ui-capture-video.png"]
+
+.Example
+[source,javascript]
+----
+  const captureVideo = async () => {
+    const capturedVideo = await Koji.ui.capture.video({ hideExtensions : false });
+    if (capturedVideo) alert( 'You chose/uploaded a video file: ' + capturedVideo );
+  }
+  ...
+  return (
+    <div className="App">
+      <header className="App-header" style={{ backgroundColor }}>
+        <button onClick={captureVideo}>{'.video test'}</button>
+      </header>
+    </div>
+  );
+----
+====
+
+[TIP]
+====
+The @withkoji/core package provides multiple APIs for uploading files.
+If your app needs a specific type of file (audio, image, or video), use the specific API for that file type if possible.
+
+That is, instead of `.media` or `.file`,
+
+* To accept only audio files, use `.audio`.
+* To accept only images, use `.image`.
+* To accept only videos, use `.video`.
+
+The specific APIs help to filter the selections offered to the user or to validate the uploaded file, if any.
+====
+
+== Related resources
+
+* Koji App Store: https://withkoji.com/apps/67dea854-70b6-4114-98ad-1e28a60d6fb0[Example | Capturing User Input]
+* <<core-frontend-ui-capture#, Frontend / UI / Capture>>
+* <<core-frontend-ui-present#, Frontend / UI / Present>>
```

Comment:
We don't discuss present on the page, so let's remove it. Instead, we can add upload if we discuss that in the file section.

Reply:
Removed.